docs/network-policy.md

```diff
@@ -0,0 +1,365 @@
+# NetworkPolicy
+
+Kubernetes NetworkPolicy documentation: https://kubernetes.io/docs/concepts/services-networking/network-policies
+
+Kubernetes NetworkPolicy API reference: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#networkpolicy-v1-networking-k8s-io
+
+By default K8's pods may send to any recipient and receive from any source. When a pod is selected by one or more NetworkPolicies it becomes isolated and all traffic is blocked. NetworkPolicy may then be used to allow received traffic from sources determined by the `ingress` section and/or send traffic to recipients specified by the `egress` section.
```

Comment:
I think this is awkwardly worded, maybe something like 

"By default the network traffic from and to K8s pods are not restricted in any way. Using Network Policy is a way to enforce network isolation of selected pods. When a pod is selected by a NetworkPolicy recived traffic is determined by the `ingress`section and/or sent traffic is specified by the `egress` section

Reply:
Agreed, I became an ENG so that I wouldn't have to write :)